android/src/main/java/com/audiowaveform/AudioWaveformModule.kt

```diff
@@ -100,6 +101,10 @@ class AudioWaveformModule(context: ReactApplicationContext): ReactContextBaseJav
         obj: ReadableMap,
         promise: Promise
     ) {
+        if(audioPlayers.filter { it.value?.isHoldingAudioTrack() == true }.count() >= MAX_NUMBER_OF_AUDIO_PLAYER) {
+            promise.reject(Constants.LOG_TAG, "Too many players have been initialized. Please stop some players before continuing")
+        }
```

Comment:
@nilesh-simform  Have we tested this more than 30 or 40 audios? 
How does this work? Can you please explain bit more here.

Is this issue for Android only?

Reply:
Yes, the issue occurs only on Android because it has a limit of 32 audio tack object. When this limit is exceeded, it stops working. To fix this, we are clearing the audio instance when the player is stopped. I tested it by playing and stopping multiple audios, and I was able to reproduce the error before. Now, with this fix, it works fine.